readme.md

```diff
@@ -1,60 +1,35 @@
 # internships
-
 Website for apprenticeship programme.
-
-## Usage & Development
-
-### 1. Create `.env` file based on `.env.example`:
-
+# Architecture
+The project contains a modified default configuration and is structured as follows:
+* `backend` stores configuration for Composer and Laravel application and other PHP related stuff;
+* `frontend` is for Node.js npm, Vite, Vue.js and lint configuration - it also contains assets used in the webpage;
+
+## Setup and Run
+### 1. Run setup script:
+The script will ask you to input a MapBox public key, which will be written into a .env file as VITE_MAPBOX_TOKEN:
```

Comment:
Now script isn't asking - it's just friendly remainder on the end to after setting variables in .env file to restart containers.